examples/jsm/nodes/utils/ColorSpaceNode.d.ts

```diff
@@ -0,0 +1,57 @@
+import { Node } from '../core/Node';
+import { TempNode } from '../core/TempNode';
+import { FunctionNode } from '../core/FunctionNode';
+import { NodeBuilder } from '../core/NodeBuilder';
+
+export class ColorSpaceNode extends TempNode {
+  constructor(input: Node, method?: string);
+
+  input: Node;
+  method: string | undefined;
+  nodeType: string;
+
+  generate(builder: NodeBuilder, output: string): string;
+  fromEncoding(encoding: number);
+  fromDecoding(encoding: number);
```

Comment:
I guess the parameter of `fromDecoding()` should be actually `decoding`?

Reply:
I am not sure
this is almost a fork of:
https://github.com/mrdoob/three.js/blob/add00a727b781b3ae1632f52108332edfc28680a/src/renderers/webgl/WebGLProgram.js#L67

And it should use `TextureEncoding`  as input
https://github.com/mrdoob/three.js/blob/5403a8a793e2a64bff1e1f2009f0a120d1db9b48/src/textures/Texture.d.ts#L27

Reply:
Okay, then let's keep it for now.